biojava-structure/src/main/java/org/biojava/nbio/structure/secstruc/SecStrucCalc.java

```diff
@@ -679,6 +680,14 @@ public String printFASTA() {
 	}
 
 	@Override
+	public int hashCode() {
+	    final int prime = 31;
+	    int result = 1;
+	    result = prime * result + Arrays.hashCode(atoms);
+	    return result;
+	}
+
+	@Override
```

Comment:
This is not acting on the same fields as the `equals` below (`equals` checks for sec structure elements to be the same).

The comment is a bit late, now that it's been merged. But we should try to solve this...